From 51ec4c1f533dee9a499490c1c5bbd23d2191a4a7 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 20 Mar 2022 11:14:25 -0700
Subject: [PATCH] cmake: Use CMAKE_INSTALL_LIBDIR

Do not hardcode library path, this helps it install in right location
independent of platforms, e.g. ppc64 uses lib64 and not lib

Upstream-Status: Submitted [https://github.com/mfontanini/cppkafka/pull/291]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c4e44e6..25ec1ce 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -51,7 +51,7 @@ option(CPPKAFKA_EXPORT_CMAKE_CONFIG "Generate CMake config, target and version f
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/")
 
 if (NOT CPPKAFKA_CONFIG_DIR)
-    set(CPPKAFKA_CONFIG_DIR lib/cmake/${PROJECT_NAME})
+    set(CPPKAFKA_CONFIG_DIR ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
 endif()
 
 # Maintain previous compatibility
-- 
2.35.1

